pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Comment:
For once I'm good with this being Pre-alpha

Reply:
I had the same thought